lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -46,10 +46,7 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
           return true;
         }
       })
-      .filter(anchor => {
-        // Ignore href's that are not real links
-        return !anchor.href || !anchor.href.toLowerCase().startsWith('javascript:');
-      })
+      .filter(anchor => anchor.href.toLowerCase().startsWith('http'))
```

Comment:
probably need a `anchor.href &&` here to be safe

Reply:
@patrickhulce I meant to ask about that, it was previously checking for `!anchor.href`, and the unit tests seem to suggest that's what it's wanting to do, so I've put that back in.

https://github.com/GoogleChrome/lighthouse/blob/f49ef0602537de1a87e692960d8962974c566e1e/lighthouse-core/test/audits/dobetterweb/external-anchors-use-rel-noopener-test.js#L43-L49

Reply:
Unless there is an error with this unit test and this audit should ignore anchors without external links, and I presume you need an href to be an external link.

Reply:
oh whoops, yeah that's right you can ignore my comment then :) 

when glancing I assumed it was there as a `null` guard for the next case, but I remember now it was intentional